src/scene_battle.cpp

```diff
@@ -159,9 +159,28 @@ void Scene_Battle::DrawBackground(Bitmap& dst) {
 
 void Scene_Battle::CreateUi() {
 	std::vector<std::string> commands;
-	commands.push_back(ToString(lcf::Data::terms.battle_fight));
-	commands.push_back(ToString(lcf::Data::terms.battle_auto));
-	commands.push_back(ToString(lcf::Data::terms.battle_escape));
+
+	for (std::vector<int16_t>::iterator it = lcf::Data::system.easyrpg_battle_options.begin();
```

Comment:
this can be replaced with "for (auto option: easyrpg_battle_options)"

Same for "All all menu items" below

Reply:
Fixed.